src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,11 @@
+package course_project;
+
+import course_project.services.GameService;
+
+public class Main {
+
+    public static void main(String[] args) {
+        new GameService().run();
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/134034072-169a987c-579a-45d4-b903-c5176772730d.png)

Reply:
I really don't understand what happened. It seems the interface needs refactoring. Also, it would be better if you add visual field representation

Reply:
There's much repetitive code, and it makes reading code harder

Reply:
![image](https://user-images.githubusercontent.com/58872509/134035573-4ee037c9-73ed-45ca-bca8-3cbb78572f53.png)

Reply:
I've tried to write in detail what user should do. I didn't add visual field representation as in the game opponents are not supposed to see each other's fields. So I am not sure that any visual representation would be appropriate here. Although, in the case of computer win user can see computer ship's location.

Reply:
Here a bug again... 
![image](https://user-images.githubusercontent.com/58872509/134490076-b9e07deb-61be-4977-8493-62bb423fbba1.png)

Reply:
But the interface was refactored, although I couldn't continue to play

Reply:
Good, I didn't find any bugs! As optional improvement, marking cells around a sunk ship as hit is a good idea. Approved!